src/test/java/picard/sam/testers/SamFileTester.java

```diff
@@ -364,10 +373,25 @@ public void runTest() {
         }
     }
 
-    private File createInputFile() {
+    private File createInputFile(final String extension) throws IOException {
         // Create the input file
-        final File input = new File(outputDir, "input.sam");
-        final SAMFileWriter writer = new SAMFileWriterFactory().makeSAMOrBAMWriter(samRecordSetBuilder.getHeader(), true, input);
+        final File input = new File(outputDir, "input" + extension);
+
+        final SAMFileWriterFactory samFileWriterFactory = new SAMFileWriterFactory();
+
+        final SAMFileWriter writer;
+        if(extension.equals(".cram")) {
+            final Path fasta = IOUtil.getPath("Testinput.fasta");
```

Comment:
Why not `Paths.get()`?
```suggestion
            final Path fasta = Paths.get("Testinput.fasta");
```

I think you want to create a file in a temp directory here, not in the source tree. E.g. `Files.createTempFile("testinput", ".fasta");`